### ISSUE TITLE
Need Update to Support Low end PC

### ISSUE THREAD

**Issue by Tri44** [citra-emu/citra#3774]
please make citra emulator can run at least in OpenGL 3.1/3.0
I'll wait that Update anytime.

**Comment by wwylele** [comment]
Here is a list the feature we are using from OpenGL 3.2/3.3
 - Sampler Objects: significantly simplify the code logic around texture/sampler binding. Removing this would result in a large amount of texture state setting/resetting boilerplate and runtime overhead of more gl calls.
 - Drawing command allowing modification of the base vertex index (`glDrawElementsBaseVertex` etc) this is the what is currently used for indexed drawing with hardware shader. Without this one would need to either always upload the vertex buffer from the beginning, or modify the indices in the index buffer manually. Either way would result in overhead
 - Geometry Shaders: an essential part to make hardware shader work

Basically, to support OpenGL 3.1, the other 90% users using >3.3 would be hurted.

**Comment by Hexagon12** [comment]
GitHub isn't used for support requests. If you have any questions for support, take them to our forums.

Requesting to close.
